html/html-and-the-web_div-salad/index.html

```diff
@@ -12,60 +12,70 @@
     <meta charset="UTF-8" />
   </head>
   <!--someone forgot to wrap the content of our webpage with an HTML element,
-	which one was it again?-->
-  <div>
-    <div>
-      <div>Jane Doe</div>
-      <!--Which elements could be used to link to parts of the webpage,
-		which appear below?-->
-      <div id="nav">
-        <div>About Me</div>
-        <div>Contact</div>
-        <div>Skillset</div>
-      </div>
+	which one was it again?   DONE -->
+  <header>
+    <div><h1>Jane Doe</h1></div>
+    <!--Which elements could be used to link to parts of the webpage,
+		which appear below?  DONE -->
+    <div id="nav">
```

Comment:
the challenge is to replace `div` with semantic tags like `section`, `main`, `header` etc.